lib/decorators/names_and_party.rb

```diff
@@ -0,0 +1,20 @@
+require 'scraped'
+
+class NamesAndParty < Scraped::Response::Decorator
+  def body
+    doc = Nokogiri::HTML(super)
+    doc.xpath('//div[@class="entry"]/table[1]/tbody/tr/td[2]').each do |member_row|
+      name_and_party = member_row.text.split(/\s(?=\()/)
+      party = Nokogiri::XML::Node.new 'span', doc
+      party.content = name_and_party.last
+      party['class'] = 'ep_member_party'
+      name = Nokogiri::XML::Node.new 'span', doc
+      name.content = name_and_party.first
+      name['class'] = 'ep_member_name'
+      member_row.content = ''
+      member_row.add_child(name)
+      member_row.add_child(party)
+    end
+    doc.to_s
+  end
```

Comment:
I'm finding this very hard to review, I'm afraid. I can't really tell from the code what it's doing, or why. At the very least, I think the class needs a better name, that reveals its intention better. As this is a new class, it should also pass `rubocop` checking, which will also require simplifying it, or breaking it up into smaller, more understandable pieces.

However… I'm a little suspicious that this is even the right approach in the first place. If you're going to pre-process this as a decorator, wouldn't it be cleaner to turn these into separate columns, rather than just wrapping them in spans? Though, either way, I fear that this is pushing the limits of a simple decorator, and this should really just be part of the logic of the Section instead.